packages/auth/src/graphql/schema.graphql

```diff
@@ -16,10 +16,7 @@ type Query {
   ): GrantsConnection!
 
   "Fetch a specific grant by its ID."
-  grant(
-    "Unique identifier of the grant."
-    id: ID!
-  ): Grant!
+  grant("Unique identifier of the grant." id: ID!): Grant!
```

Comment:
some formatting issue made it into main somehow

Reply:
If you merge in main this should go away